### DISCUSSION TITLE
In a new open dataset,the performance of the tsm is poor.Which lead to this?

### DISCUSSION THREAD

**Discussion by ZJU-lishuang** [open-mmlab/mmaction2#631] [General] [answered]
I train the tsm on the [fight dataset](https://github.com/sayibet/fight-detection-surv-dataset).
And the best top1 acc is 0.53.
When I use other [small open dataset](https://www.openu.ac.il/home/hassner/data/violentflows/),the top1 acc is nearly to 1.0.
Is the [fight dataset](https://github.com/sayibet/fight-detection-surv-dataset) problem lead to the problem?

**Comment by ZJU-lishuang** [comment]
config
[tsm_r50_1x1x8_50e_customdataset_fight_rgb320.py](https://github.com/open-mmlab/mmaction2/files/6026639/tsm_r50_1x1x8_50e_customdataset_fight_rgb320.txt)
log
[20210220_081407.log](https://github.com/open-mmlab/mmaction2/files/6026641/20210220_081407.log)

Can you help me?

**Comment by innerlee** [comment]
try i3d?

**Comment by innerlee** [comment]
And use pre-trained model on k400

**Comment by ZJU-lishuang** [comment]
Oh,I forget to use pre-trained model on k400.
I will have a try.

**Comment by ZJU-lishuang** [comment]
@innerlee When I do as you said ,the top1 acc of last model is higher than past.
But the top1 acc value is smaller and smaller as the training continues.
i3d log with k400 pre-train
[20210223_092126.log](https://github.com/open-mmlab/mmaction2/files/6028192/20210223_092126.log)

tsm log with k400 pre-train
[20210223_081901.log](https://github.com/open-mmlab/mmaction2/files/6028012/20210223_081901.log)

Is the dataset problem leading to low top1 acc?

**Comment by innerlee** [comment]
The dataset is very tiny. 

* Use small lr. Divide the original lr by 10 or 100
* Use strong augmentation. See imgaug examples https://github.com/open-mmlab/mmaction2/pull/492

**Reply by ZJU-lishuang** [reply]
because of the small dataset.
These two strategies are useless.

**Comment by innerlee** [comment]
* Fix the first two stages of the backbone

**Reply by ZJU-lishuang** [reply]
@innerlee How to do it?
There isn't a tutorial to Freeze Backbone.
Is The relative parameter `frozen_stages`? I don't know how to use it.
Set `frozen_stages=2`?

**Comment by irvingzhang0512** [comment]
It's overfitting. For small datasets, fewer epochs (i.e. 20) are enough.
You can also set `interval=1` for both `evaluation` and `checkpoint_config` in your configs.

**Reply by ZJU-lishuang** [reply]
early stop is a good idea.

**Comment by ZJU-lishuang** [comment, marked as answer]
using pre-trained model on k400 and early stop get a good result.
